addons/stock/models/stock_orderpoint.py

```diff
@@ -184,6 +189,16 @@ def _onchange_route_id(self):
         if self.route_id:
             self.qty_multiple = self._get_qty_multiple_to_order()
 
+    @api.onchange('product_min_qty')
+    def _onchange_min_qty(self):
+        if self.product_min_qty > self.product_max_qty:
+            self.product_max_qty = self.product_min_qty
+
+    @api.onchange('product_max_qty')
+    def _onchange_max_qty(self):
```

Comment:
Couldn't be better to replace those onchanges by computes ?
That way, the same logic will be applied when modifying records directly (in tests for example), making easier to modify the `product_min_qty`/`product_max_qty`

Reply:
Yes, makes sense 👍